dialogue-pizzaconundrum.js

```diff
@@ -25,7 +25,7 @@ const PIZZAMOOD_OMG = [
     segments: [
       "coolio: 🍍🍕 🤮",
       "77gus77: pizza toppings a bid deal! 🧐",
-      "bibop: 🍍 or not 🍍 that is the ❔",
+      "bibop: 🍍 or not 🍍 that is the ❓",
```

Comment:
The white question mark didn't have enough contrast for me to see it
![Screen Shot 2022-02-05 at 11 31 21 AM](https://user-images.githubusercontent.com/21364548/152650399-7053b256-2ca8-4e83-87f6-134abc8dcade.png)
 Suggesting replacing with the red one instead